easybuild/main.py

```diff
@@ -70,6 +70,7 @@
 from easybuild.tools.parallelbuild import submit_jobs
 from easybuild.tools.repository.repository import init_repository
 from easybuild.tools.testing import create_test_report, overall_test_report, regtest, session_state
+from easybuild.tools.easystack import parse_easystack
```

Comment:
@deniskristak I think it makes sense to move `easybuild.tools.easystack` to `easybuild.framework.easystack` (since also the code to deal with easyconfigs and easyblocks lives there).

Reply:
done